From 828ac5f5a392b68ff85ded56562bda20cf4fa8d8 Mon Sep 17 00:00:00 2001
From: "shaikh.kamal" <shaikhkamal2012@gmail.com>
Date: Sun, 26 Jul 2026 12:28:35 +0530
Subject: [PATCH 12898/15333] docs: hwmon: sg2042-mcu: fix spelling error

Fix spelling error reported by codespell:

  supprts -> supports

No functional change.

Signed-off-by: shaikh.kamal <shaikhkamal2012@gmail.com>
Link: https://lore.kernel.org/r/20260726065835.21012-1-shaikhkamal2012@gmail.com
[groeck: Move driver name to subject line]
Signed-off-by: Guenter Roeck <linux@roeck-us.net>
---
 Documentation/hwmon/sg2042-mcu.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/hwmon/sg2042-mcu.rst b/Documentation/hwmon/sg2042-mcu.rst
index 077e79841d2e..431ed832d083 100644
--- a/Documentation/hwmon/sg2042-mcu.rst
+++ b/Documentation/hwmon/sg2042-mcu.rst
@@ -18,7 +18,7 @@ Authors:
 Description
 -----------
 
-This driver supprts hardware monitoring for onboard MCU with
+This driver supports hardware monitoring for onboard MCU with
 i2c interface.
 
 Usage Notes
-- 
2.55.0

